### PR TITLE
Metadata updates

### PR DESCRIPTION
**Pull request by SpaceManiac**
Changes to the readme and pom.xml for Glowkit, looking for review.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-        <url>https://github.com/Bukkit/Bukkit/tree/master/</url>
+        <developerConnection>scm:git:git@github.com:GlowstoneMC/Glowkit.git</developerConnection>
+        <connection>scm:git:git://github.com/GlowstoneMC/Glowkit.git</connection>
+        <url>https://github.com/GlowstoneMC/Glowkit/tree/master/</url>
     </scm>
 
+    <!--
     <ciManagement>
         <system>jenkins</system>
         <url>http://ci.bukkit.org</url>
```
Comment (gdude2002) [inline pom.xml:22]: _cough_

File: pom.xml [hunk 2]
```diff
@@ -37,6 +38,7 @@
             <url>http://repo.bukkit.org/content/repositories/snapshots/</url>
```
Comment (Aaron1011) [inline pom.xml:38]: This line, also

**Comment by Tonodus** [comment]
Nice :). I always thought the bukkit api lacks some features.  I'd also clearly state that the "normal" bukkit API can be also used (outdated plugin f.e.) if thats true.

File: pom.xml (hunk 1 above)
Comment (SpaceManiac) [inline pom.xml:22]: note that this segment is commented out.

**Pull request closed without merging**